imagepipeline/src/main/java/com/facebook/imagepipeline/core/ImagePipeline.java

```diff
@@ -423,7 +423,24 @@ public boolean isInBitmapMemoryCache(final ImageRequest imageRequest) {
    * @return true if the image was found in the disk cache, false otherwise.
    */
   public boolean isInDiskCacheSync(final Uri uri) {
-    return isInDiskCacheSync(ImageRequest.fromUri(uri));
+    return isInDiskCacheSync(uri, ImageRequest.CacheChoice.DEFAULT);
```

Comment:
If I followed code correctly, `isInDiskCacheSync` method finally check disk cache with file check.

If you see `diskCheckSync` method with this [link](https://github.com/facebook/fresco/blob/master/imagepipeline/src/main/java/com/facebook/imagepipeline/cache/BufferedDiskCache.java#L113-L123) in `BufferedDiskCache`, that code is double checking in-memory index and file cache.

Because of calling ` checkInStagingAreaAndFileCache(CacheKey)`, it will check file cache by [this line](https://github.com/facebook/fresco/blob/master/imagepipeline/src/main/java/com/facebook/imagepipeline/cache/BufferedDiskCache.java#L158) if there is no index in in-memory index list.

So I thought, if there are someone who using this method currently, this change would make different result and more slow performance. So I just make it to check `DEFAULT`  disk cache if no cache-type provided.

Is there any problem with that?